midend/actionSynthesis.cpp

```diff
@@ -191,7 +193,7 @@ const IR::Statement* DoSynthesizeActions::createAction(const IR::Statement* toAd
     actions.push_back(action);
     auto actpath = new IR::PathExpression(name);
     auto repl = new IR::MethodCallExpression(actpath);
-    auto result = new IR::MethodCallStatement(repl->srcInfo, repl);
+    auto result = new IR::MethodCallStatement(toAdd->srcInfo, repl);
```

Comment:
Maybe the expression above should also use toAdd->srcInfo.

Reply:
makes sense